app/controllers/buyers/applications_controller.rb

```diff
@@ -6,8 +8,9 @@ class Buyers::ApplicationsController < FrontendController
   helper DisplayViewPortion::Helper
 
   before_action :authorize_partners
-  before_action :find_buyer, :only => [:new, :create]
-  before_action :authorize_multiple_applications, :only => [ :new, :create ]
+  before_action :find_buyer, :only => [:create]
```

Comment:
Why not `:new`?

Reply:
Because now buyer is not always available and it will return 404.